Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreterEmitter.cpp

```diff
@@ -36,6 +36,5 @@ void CachedInterpreterCodeBlock::PoisonMemory()
 {
   DEBUG_ASSERT(reinterpret_cast<std::uintptr_t>(region) % alignof(AnyCallback) == 0);
   DEBUG_ASSERT(region_size % sizeof(AnyCallback) == 0);
-  std::fill(reinterpret_cast<AnyCallback*>(region),
-            reinterpret_cast<AnyCallback*>(region + region_size), AnyCallbackCast(PoisonCallback));
+  std::fill_n(reinterpret_cast<AnyCallback*>(region), region_size, AnyCallbackCast(PoisonCallback));
```

Comment:
Shouldn't this be replaced by the following instead?
```suggestion
  std::fill_n(reinterpret_cast<AnyCallback*>(region), region_size / sizeof(AnyCallback), AnyCallbackCast(PoisonCallback));
```

IIRC, `(AnyCallback*)(region + size)` might not be equal to `(AnyCallback*)(region) + size`.

Especially when `sizeof(region) != sizeof(AnyCallback)` since `(T*)(ptr) + 1` will increment `ptr` of `sizeof(T)` bytes.

Reply:
Wow, this is *my own code* I've changed here, yet I left in such an obvious mistake. Maybe I was a bit hasty when uploading these drafts for initial feedback...

Reply:
Rather than dividing `region_size` by `sizeof(AnyCallback)`, I have reverted this change.